.circleci/config.yml

```diff
@@ -6,24 +6,34 @@ version: 2.1  # Version of CircleCI config format
 orbs:
   codecov: codecov/codecov@1.1.0  # support for uploading code coverage to codecov
 
+defaults: &defaults
+  docker:
+    - image: humancompatibleai/seals:base
+      auth:
+        username: $DOCKERHUB_USERNAME
+        password: $DOCKERHUB_PASSWORD
+  working_directory: /seals
```

Comment:
I'm okay with this, but curious why you introduced a default dict given that it's only used once in this config. Maybe for reduced nesting or standardized config.yml form?